src/cc65/declare.c

```diff
@@ -771,17 +771,19 @@ static SymEntry* ParseStructDecl (const char* Name)
             */
             if (BitOffs > 0) {
                 if (FieldWidth <= 0 || (BitOffs + FieldWidth) > INT_BITS) {
+                    /* Bits needed to byte-align the next field. */
+                    unsigned PaddingBitWidth = -BitOffs % CHAR_BITS;
```

Comment:
I think that it should be
```c
unsigned int PaddingBitWidth = CHAR_BITS - (BitOffs % CHAR_BITS);
```

Reply:
If `BitOffs` is 8 or 16, we want 0 bits of padding, not 8.